### PR TITLE
without_conda

### PR DESCRIPTION
**Pull request by Wangwuyi123**
## How Has This Been Tested?
- [ ] Pass the test by running: `pytest qlib/tests/test_all_pipeline.py` under upper directory of `qlib`.
- [ ] If you are adding a new feature, test on your own test scripts.

## Screenshots of Test Results (if appropriate):
1. Pipeline test:
2. Your own tests:

## Types of changes

- [ ] Fix bugs
- [ ] Add new feature
- [ ] Update documentation

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest, ubuntu-16.04, ubuntu-18.04, ubuntu-20.04]
```
Comment (zhupr) [inline .github/workflows/test.yml:15]: add `macos-latest` :
`os: [windows-latest, ubuntu-16.04, ubuntu-18.04, ubuntu-20.04, macos-latest]`

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -12,8 +12,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [windows-latest, ubuntu-16.04, ubuntu-18.04, ubuntu-20.04]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        os: [windows-latest, ubuntu-16.04, ubuntu-18.04, ubuntu-20.04，macos-latest]
```
Comment (zhupr) [inline .github/workflows/test.yml:15]: At the end write a `","` you write the Chinese symbol `"，"`
```python
[windows-latest, ubuntu-16.04, ubuntu-18.04, ubuntu-20.04，macos-latest]
```

File: .idea/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (zhupr) [inline .idea/.gitignore:1]: delete `.idea`

File: .github/workflows/test.yml (hunk 1 above)
Comment (Wangwuyi123) [inline .github/workflows/test.yml:15]: There are some issues (in the downloading data phase) on MacOS when running with other tests. So we split it into an individual config.

**Pull request merged**